### PR TITLE
cleaning up the first few paragraphs of getting started with containers

### PR DESCRIPTION
**Pull request by mikebrow**
Updated docs to address issue #16161 

Signed-off-by: Mike Brown brownwm@us.ibm.com

### CODE REVIEW

File: docs/articles/basics.md [hunk 1]
```diff
@@ -19,12 +19,14 @@ installed, use the following command:
 If you get `docker: command not found` or something like
 `/var/lib/docker/repositories: permission denied` you may have an
 incomplete Docker installation or insufficient privileges to access
-Docker on your machine. Please 
+Docker on your machine. With the default installation of Docker, `docker`
+commands need to be run by at least one of a user that is in the `docker` group
```
Comment (thaJeztah) [inline docs/articles/basics.md]: I'd drop "at least one of" here

File: docs/articles/basics.md [hunk 2]
```diff
@@ -19,12 +19,14 @@ installed, use the following command:
...
+or by the `root` user.           
 
-Additionally, depending on your Docker system configuration, you may be required
-to preface each `docker` command with `sudo`. To avoid having to use `sudo` with
-the `docker` command, your system administrator can create a Unix group called
-`docker` and add users to it.
+For example, depending on your Docker system configuration, you may be required
```
Comment (thaJeztah) [inline docs/articles/basics.md]: You can remove "For example," here, I don't think it adds much :-)

File: docs/articles/basics.md [hunk 3]
```diff
@@ -53,9 +55,13 @@ To run an interactive shell in the Ubuntu image:
...
     $ docker run -i -t ubuntu /bin/bash       
   
-The `-i` flag starts an interactive container. The `-t` flag creates a pseudo-TTY that attaches `stdin` and `stdout`.  
+The `-i` flag starts an interactive container. The `-t` flag creates a
+pseudo-TTY that attaches `stdin` and `stdout`.  
 
-To detach the `tty` without exiting the shell, use the escape sequence `Ctrl-p` + `Ctrl-q`. The container will continue to exist in a stopped state once exited. To list all containers, stopped and running use the `docker ps -a` command.
+To detach the `tty` without exiting the shell, use the escape sequence
+`Ctrl-p` + `Ctrl-q`. The container will continue to exist in a stopped state
+once exited. To list all containers, stopped and running use the `docker ps -a`
```
Comment (thaJeztah) [inline docs/articles/basics.md]: Reading this, I think there should be a comma after "stopped and running" (not your change, just noticed it here)

**Comment by thaJeztah** [comment]
Thanks again! Added some suggestions, but looking good.

The `sudo` stuff has always been a "going back and forth" change, because it really depends on the setup (e.g., an OS X user, running only a docker *client* shouldn't use `sudo`). This looks good to me though (fingers crossed :smile:)

**Comment by mikebrow** [comment]
@thaJeztah thx for the review.  I made the changes you recommended.  

Cheers, Mike

**Comment by thaJeztah** [comment]
Thanks! LGTM

**Comment by thaJeztah** [comment]
Oh! Overlooked that; could you squash your commits? :relaxed:

**Comment by mikebrow** [comment]
Squashed... and putting a yellow sticky on my monitor.. make squash :-)

**Comment by thaJeztah** [comment]
Hehe, great!

File: docs/articles/basics.md [hunk 4]
```diff
@@ -10,21 +10,22 @@ parent = "smn_containers"
...
+With the default installation of Docker, `docker` commands need to be run by a
+user that is in the `docker` group or by the `root` user.           
 
-Additionally, depending on your Docker system configuration, you may be required
-to preface each `docker` command with `sudo`. To avoid having to use `sudo` with
-the `docker` command, your system administrator can create a Unix group called
-`docker` and add users to it.
+Depending on your Docker system configuration, you may be required to preface
+each `docker` command with `sudo`. One way to avoid having to use `sudo` with
+`docker` commands, is to create a Unix group called `docker` and add users that
```
Comment (MHBauer) [inline docs/articles/basics.md]: nit, It's all one sentence, I don't think there needs to be a comma.

File: docs/articles/basics.md [hunk 5]
```diff
@@ -53,9 +54,13 @@ To run an interactive shell in the Ubuntu image:
...
     $ docker run -i -t ubuntu /bin/bash       
   
-The `-i` flag starts an interactive container. The `-t` flag creates a pseudo-TTY that attaches `stdin` and `stdout`.  
+The `-i` flag starts an interactive container. The `-t` flag creates a
+pseudo-TTY that attaches `stdin` and `stdout`.  
 
-To detach the `tty` without exiting the shell, use the escape sequence `Ctrl-p` + `Ctrl-q`. The container will continue to exist in a stopped state once exited. To list all containers, stopped and running use the `docker ps -a` command.
+To detach the `tty` without exiting the shell, use the escape sequence
+`Ctrl-p` + `Ctrl-q`. The container will continue to exist in a stopped state
+once exited. To list all containers, stopped and running, use the `docker ps -a`
```
Comment (MHBauer) [inline docs/articles/basics.md]: good comma.

**Comment by MHBauer** [comment]
I like the it too, for what it's worth.

**Comment by thaJeztah** [comment]
ping @fredlf @jamtur01 @moxiegirl @SvenDowideit

**Comment by moxiegirl** [comment]
Thanks @mikebrow LGTM.

**Pull request merged**